src/utils/OnboardingProvider.tsx

```diff
@@ -114,15 +103,8 @@ export default function OnboardingProvider({
           );
       }
 
-      if (canReads.size === 0)
-        throw new Error(
-          'Error flushing data: can read languages data is empty!',
-        );
-
-      if (canSpeaks.size === 0)
-        throw new Error(
-          'Error flushing data: can write languages data is empty!',
-        );
+      if (canReads.length + canSpeaks.length === 0)
```

Comment:
do we want to error when either "can read" or "can speak" is left blank, rather than erroring only when both are blank?

Reply:
i think either, it could be the case that one person can speak but cannot read/speak